packages/web-react/tests/testUtils/componentAccessibilityTest.ts

```diff
@@ -0,0 +1,25 @@
+import { type render } from '@testing-library/react';
+import { type RunAxeOptions, runAxe } from './runAxe';
+
+interface A11yScenario {
```

Comment:
```suggestion
interface AccessibilityScenario {
```

I suggest using `accessibility` or `a11y` for all usage, but not mixing both.